node/p2p/bitswap.go

```diff
@@ -20,7 +20,14 @@ import (
 // DataExchange provides a constructor for IPFS block's DataExchange over BitSwap.
 func DataExchange(cfg Config) func(bitSwapParams) (exchange.Interface, blockstore.Blockstore, error) {
 	return func(params bitSwapParams) (exchange.Interface, blockstore.Blockstore, error) {
-		bs := blockstore.NewBlockstore(params.Ds)
+		bs, err := blockstore.CachedBlockstore(
+			fxutil.WithLifecycle(params.Ctx, params.Lc),
+			blockstore.NewBlockstore(params.Ds),
+			blockstore.DefaultCacheOpts(),
```

Comment:
Let's copy those defaults at the top of the file for visibility and as a reminder that we might tune those in the future.